setup.py

```diff
@@ -205,6 +207,7 @@ def get_requirements():
         'pptk',
```

Comment:
AFAIK, `pptk` newer versions have been causing viz errors as well. Possibly in a later PR, let's identify a stable version of `pptk` that builds and works w/o hassle.

Reply:
Will do it in a subsequent PR